install_pim/system_requirements/manual_system_installation_debian9.rst

```diff
@@ -0,0 +1,85 @@
+Manual System Installation on Debian 9 Stretch
+==============================================
+
+Here is a quick guide to set up the :doc:`system_requirements` manually on Debian 9 Stretch.
+
+.. note::
+
+    Please perform the following commands as root.
+
+System installation
+-------------------
+
+MySQL 5.7
+*********
+
+The easiest way to install MySQL 5.7 is to use the official vendor package.
+
+First, start by installing the `MySQL APT repository <https://dev.mysql.com/doc/mysql-apt-repo-quick-guide/en/#apt-repo-setup>`_.
+It's a tool that aims to ease the installation and update of MySQL products.
```

Comment:
too late :p but can you finish your review please? I'll do a pr later to address your comments